modules/bbmap/index/main.nf

```diff
@@ -0,0 +1,41 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+def VERSION = '38.92'       // There seems to be no way to get the version out of the bbmap.sh script, only a line with "Last modified" and a date
```

Comment:
See here how to do it: https://github.com/nf-core/modules/blob/master/modules/bbmap/bbduk/main.nf

Reply:
Thanks, didn't know that.